torchgeo/models/unet.py

```diff
@@ -39,6 +39,46 @@ class Unet_Weights(WeightsEnum):  # type: ignore[misc]
     .. versionadded:: 0.8
     """
 
+    UMBRA_GEC_OPENEARTHMAP_SAR = Weights(
+        url='https://hf.co/torchgeo/umbra_gec_unet_effb4_openearthmap_sar/resolve/738e0216fa5d41d14f535cc28430052683704142/umbra_gec_unet_effb4_openearthmap_sar-f21df2ee.pth',
+        transforms=K.AugmentationSequential(
+            K.Normalize(mean=torch.tensor(0), std=torch.tensor(255)), data_keys=None
+        ),
+        meta={
+            'dataset': 'OpenEarthMap-SAR',
+            'in_chans': 1,
+            'num_classes': 9,
+            'model': 'U-Net',
+            'encoder': 'efficientnet-b4',
+            'classes': (
+                'background',
+                'bareland',
+                'rangeland',
+                'developed space',
+                'road',
+                'tree',
+                'water',
+                'agriculture land',
+                'building',
+            ),
+            'colormap': (
+                '#000000',
+                '#800000',
+                '#00FF24',
+                '#949494',
+                '#FFFFFF',
+                '#226126',
+                '#0045FF',
+                '#4BB549',
+                '#DE1F07',
+            ),
+            'publication': 'https://arxiv.org/abs/2501.10891',
+            'repo': 'https://github.com/cliffbb/DFC2025-OEM-SAR-Baseline',
+            'bands': ['B1'],
```

Comment:
B1 doesn't sound right, is it VV or VH?

Reply:
It's a different level of processing that is used from Umbra where it converted to a grayscale uint8 image. This is not Sentinel-1